src/plugins/guided_onboarding/public/services/api.service.ts

```diff
@@ -113,8 +113,7 @@ export class ApiService implements GuidedOnboardingApi {
 
   /**
    * Async operation to fetch state for all guides.
-   * This is useful for the onboarding landing page,
-   * where all guides are displayed with their corresponding status.
+   * Currently only used in the example plugin.
    */
   public async fetchAllGuidesState(): Promise<{ state: GuideState[] } | undefined> {
```

Comment:
I decided to leave this function and the corresponding route because it's still used in the example plugin. This can potentially be removed in the future.